backend/config/config.go

```diff
@@ -165,8 +168,13 @@ func NewDefaultAppConfig() AppConfig {
 						PEMCert: shiftRootCA,
 					},
 				},
+				BlockExplorer: BlockExplorerConfig{
+					UseExplorer: false,
+					ExplorerURL: "https://blockstream.info/tx/",
```

Comment:
can be empty too as the coin implementation _should_ define a default. but I guess it is fine to also have the defaults here ? (hmm might be annoying for maintainability in case this is refactored later)